Makefile

```diff
@@ -260,7 +260,7 @@ requirements.txt:
 	if [ ! -d ".git" ]; then git init; fi
 	rm -rf .venv/git-secrets
 	git clone https://github.com/awslabs/git-secrets .venv/git-secrets
-	cd .venv/git-secrets && make install PREFIX=..
+	cd .venv/git-secrets  && git reset --hard 635895a8d1b7c976ac9794cef420f8dc111a24d4 && make install PREFIX=..
```

Comment:
otherwise hooks don't work anymore, it's a cherrypick of this PR on `chsdi` https://github.com/geoadmin/mf-chsdi3/commit/5d4515b1ec0eb46092412ac40ec0d2540132e857#diff-b67911656ef5d18c4ae36cb6741b7965 (as this is some kind of a rip off of chsdi, this was also needed)